pkg/trace/agent/agent.go

```diff
@@ -58,8 +58,8 @@ func NewAgent(ctx context.Context, conf *config.AgentConfig) *Agent {
 	dynConf := sampler.NewDynamicConfig(conf.DefaultEnv)
 
 	// inter-component channels
-	rawTraceChan := make(chan pb.Trace, 5000) // about 1000 traces/sec for 5 sec, TODO: move to *model.Trace
-	tracePkgChan := make(chan *writer.TracePackage)
+	rawTraceChan := make(chan pb.Trace, 5000)
+	tracePkgChan := make(chan *writer.TracePackage, 1000)
```

Comment:
Suggestion: could be useful to regularly send stats for the number of items queued in all of these channels, to help track down where/when things get backed up.